easybuild/easyconfigs/c/Chemaxon_Marvin/Chemaxon_Marvin-21.14.eb

```diff
@@ -0,0 +1,69 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+
+# rpm file extension does not look to be known yet: Unknown file type for file *.rpm (at easybuild/tools/filetools.py)
+# easyblock = 'Rpm'
+easyblock = 'Binary'
+
+name = 'Chemaxon_Marvin'
+version = '21.14'
+
+homepage = 'https://chemaxon.com/products/marvin'
+description = """Marvin suite is a chemically intelligent desktop toolkit built
+to help you draw, edit, publish, render, import and export your chemical structures
+and as well as allowing you to convert between various chemical and graphical file
+formats. It is free for individual, academic and non-commercial use."""
+
+# NOTE not in the license list of EB
+# software_license = 'Proprietary: Personal, Commercial or Academic'
+software_license_urls = ['https://www.chemaxon.com/marvin/license.html']
+docurls = [
+    'https://docs.chemaxon.com/display/docs/marvinsketch-user-s-guide.md',
+    'https://docs.chemaxon.com/display/docs/marvinview-user-s-guide.md',
+    'https://docs.chemaxon.com/display/docs/file-formats.md',
+]
+
+toolchain = SYSTEM
+
+sources = ['marvin_linux_%(version)s.rpm']
```

Comment:
@SIB-software Maybe include a command to mention where this can be obtained/download (even if it requires registration or a license)?

Reply:
done